catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/DataListAdapter.java

```diff
@@ -110,10 +128,25 @@ public void onBindViewHolder(CheckableViewHolder holder, int position) {
 			protected void onCheckBoxClick(int position) {
 				super.onCheckBoxClick(getRelativeItemPosition(position, VAR_GLOBAL));
 			}
+
+			@Override
+			public List<UserVariable> filterItems(List<UserVariable> all_items) {
```

Comment:
Code duplication, doesn't conform to clean code guidelines, please remove duplicated code.